050-AIS-APIManagementwithFunctions/Student/Challenge-01.md

```diff
@@ -0,0 +1,87 @@
+# Challenge 01 - Provision your Integration Environment
+
+[<Previous Challenge](./Challenge-00.md) - **[Home](../README.md)** - [Next Challenge>](./Challenge-02.md)
+
+## Pre-requisites
+
+- You should have completed Challenge 00
+
+## Introduction
+You are a cloud engineer for Runva - a global health and wellness company that provides solutions for individuals who takes running as sport, providing them a platform to track running activities, analyze performance and recommend plans to help them train smarter and safer.  They are going to embark on a six-month project to modernize their integration solution hosted on-premises and migrate them to Azure integration services.  Since this is their first foray in Azure, you are tasked to create a proof-of-concept (POC) that captures several integration patterns.
+
+
+## Description
+As a cloud engineer, you would like to be able to deploy this POC environment using Infrastructure-as-Code.  You have learned that you can efficiently do this in Azure using the new language called Bicep. As a baseline, you are going to deploy API Management service as gateway for your APIs, as well as a Function App where you would host your APIs.
+
+There are two scenarios you would like to prove (just choose one):
+
+- Scenario 1: Deploy a VNET-secured AIS environment - should you need to access services deployed over a private network (e.g. APIs hosted on-premises)
```

Comment:
For clarity, I would make each Scenario title a section sub-header using 3 hashes under the Description header.

"### Scenario 1: Deploy a VNET-secured AIS Environment"

"This scenario is for when you need to access services deployed over a private network (e.g. APIs hosted on-premises)"

- Deploy the Bicep templates...  blah blah blah...

"### Scenario 2: Deploy an Identity-Secured AIS Environment"

"This scenario is preferred for integrating with services hosed in the cloud or publicly accessible."

- Deploy the Bicep files...  blah blah blah.

Reply:
Done